src/plugins/data/public/search/expressions/__snapshots__/es_raw_response.test.ts.snap

```diff
@@ -0,0 +1,199 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`esRawResponse converts aggregations to table simple aggregation response 1`] = `
+Object {
+  "columns": Array [
+    Object {
+      "id": "2.buckets.key",
+      "meta": Object {
+        "field": "2.buckets.key",
+        "params": Object {},
+        "type": "string",
+      },
+      "name": "2.buckets.key",
+    },
+    Object {
+      "id": "2.buckets.doc_count",
+      "meta": Object {
+        "field": "2.buckets.doc_count",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.buckets.doc_count",
+    },
+    Object {
+      "id": "2.doc_count_error_upper_bound",
+      "meta": Object {
+        "field": "2.doc_count_error_upper_bound",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.doc_count_error_upper_bound",
+    },
+    Object {
+      "id": "2.sum_other_doc_count",
+      "meta": Object {
+        "field": "2.sum_other_doc_count",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.sum_other_doc_count",
+    },
+  ],
+  "meta": Object {
+    "source": "*",
+    "type": "esdsl",
+  },
+  "rows": Array [
+    Object {
+      "2.buckets.doc_count": 1033,
+      "2.buckets.key": "FEMALE",
+      "2.doc_count_error_upper_bound": 0,
+      "2.sum_other_doc_count": 0,
+    },
+    Object {
+      "2.buckets.doc_count": 944,
+      "2.buckets.key": "MALE",
+      "2.doc_count_error_upper_bound": 0,
+      "2.sum_other_doc_count": 0,
+    },
+  ],
+  "type": "datatable",
+}
+`;
+
+exports[`esRawResponse converts raw docs to table simple docs response 1`] = `
+Object {
+  "columns": Array [
+    Object {
+      "id": "order_date",
+      "meta": Object {
+        "field": "order_date",
+        "params": Object {},
+        "type": "object",
+      },
+      "name": "order_date",
+    },
+    Object {
+      "id": "products.created_on",
+      "meta": Object {
+        "field": "products.created_on",
+        "params": Object {},
+        "type": "object",
```

Comment:
This is the JS type, but it's not actually useful for our purposes. I would have expected that arrays would be flattened more.